go/enclave/storage/init/sqlite/001_init.sql

```diff
@@ -26,8 +26,8 @@ create table if not exists block
     parent       binary(32) REFERENCES block,
     is_canonical boolean,
     header       blob,
-    height       int,
-    unique (height, is_canonical)
+    height       int
+--     unique (height, is_canonical)
```

Comment:
did you mean to leave these `unique` and `references` expressions commented out?

Reply:
ah. Good point.
There is an edge case to the edge case. There could be in theory multiple non-canonical blocks/batches.
And this constraint fails because there are multiple ( 100,false) entries.

To get it to work, I'll have to do an artifice. Consider is_canonical=nil to be false, and then have the constraint work only for true.
I'll put a comment and do that as part of a different pr

Reply:
That constraint wasn't working on edgeless anyway iirc? So yeah maybe we find some other way that works in prod as well.